crates/mun_codegen/src/test.rs

```diff
@@ -106,6 +106,18 @@ fn update_operators() {
     );
 }
 
+#[test]
+fn update_parameter() {
+    test_snapshot(
+        r#"
+    fn add_three(a:int):int {
+      a += 3;
```

Comment:
To be "feature-complete" it would be good to also add `-=`, `*=`, and `/=` to the test.

Reply:
Ill add another test specifically for that

Reply:
Thats not the purpose of this test, this test just checks if updating a function parameter works in the first place. Otherwise Id have to add `+`, `-` etc too.